pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/metrics/ManagedLedgerCacheMetrics.java

```diff
@@ -48,6 +48,7 @@ public synchronized List<Metrics> generate() {
 
         m.put("brk_ml_count", mlCacheStats.getNumberOfManagedLedgers());
         m.put("brk_ml_cache_used_size", mlCacheStats.getCacheUsedSize());
+        m.put("brk_ml_cache_total_size", mlCacheStats.getCacheMaxSize());
```

Comment:
It is a fixed value and a configuration item. Why does it need to be added to the metrics?

Reply:
This configuration is dynamic, we can override it through api. 
And there so many examples like `jvm_memory_bytes_max` is also a fixed value, but exposed as a metrics.